telemetry/definitions/commonDefinitions.json

```diff
@@ -4322,6 +4330,72 @@
             ],
             "passive": true
         },
+        {
+            "name": "codewhisperer_codeScanChatFileScan",
+            "description": "Called when file scan is selected from chat",
+            "metadata": [
+                {
+                    "type": "codeScanSessionId",
```

Comment:
does this map to a service-side id? FYI, every `telemetry.xx.run(...)` "tree" of metrics automatically shares a common `traceId`: https://github.com/aws/aws-toolkit-vscode/pull/5639

Reply:
Got it, I will remove this